docs/application/native/guides/multimedia/media-handle.md

```diff
@@ -7,11 +7,11 @@ The main features of the Media Tool API include:
 
 - Creating the media format handle
 
-  You can [set and get video or audio information](#format) using the `media_format_h` handle.
+  You can [set and get video or audio information](#format) such as average bps, bit, bit depth, channel, channel mask, mime type, sample rate, width, height and frame rate for each type(audio, video or text) using the `media_format_h` handle.
```

Comment:
Just a question to (audio, video or text).
What does `text` mean in media format?

It is also said that `video` or `audio` information can be set&get, and I'm not sure what information such as bps, channel, height, and frame rate can be to text?

Reply:
The "text" means the type for "subtitle" data. If text type is set to media_format_h, all described  information such as bps, channel, sample rate and etc could not be to it.
I think it's better to remove "(audio, video or text)" to avoid confusion.
I'll update it.